frontend/food.html

```diff
@@ -18,6 +18,7 @@
                 alt="logo"
             />
             <h2 class="titleLogo">Manger Mieux</h2>
+            <a id="deco">Déconnexion</a>
```

Comment:
Good start but I think a real page to edit our parameters as a person and in this page a possibility to deconnect can be better but for now it's good.